config/config_test.go

```diff
@@ -35,3 +42,56 @@ func (s *configSuite) TestPrefixed(c *C) {
 	c.Assert(s.tlc.policy("quux"), Equals, s.tlc.prefixed(rootPolicy, "quux"))
 	c.Assert(s.tlc.volume("foo", "bar", "quux"), Equals, s.tlc.prefixed(rootVolume, "foo", "bar", "quux"))
 }
+
+func (s *configSuite) TestDumpTarball(c *C) {
+	key := "/volplugin/foo"
+	value := "baz"
+
+	exec.Command("/bin/sh", "-c", "etcdctl set "+key+" "+value).Run()
```

Comment:
After seeing the test, I figured out this would be a good idea to make it easier to test without relying on etcd.